catalyst/components/ui/enhanced-form.tsx

```
@@ -0,0 +1,552 @@
+          return 'Description must be at least 10 characters long';
+        }
+        if (typeof value === 'string' && value.length > 500) {
+          return 'Description must be less than 500 characters';
+        }
+        break;
+      
+      case 'budget':
+        if (!value || typeof value !== 'string') {
+          return 'Budget is required';
+        }
+        const budgetNum = parseFloat(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)